src/test/java/bio/terra/service/dataset/flight/inheritsteward/SetAuthGcpUserRolesStepTest.java

```diff
@@ -41,18 +41,22 @@ private void verifySetAuth(DoOrUndo doOrUndo, boolean grantPolicy) throws Except
     assertThat(doOrUndo.apply(flightContext), is(StepResult.getStepResultSuccess()));
     for (var projectId : projectIds) {
       if (grantPolicy) {
-        verify(resourceService).grantPoliciesBqJobUser(projectId, List.of(custodianEmail));
+        verify(resourceService)
+            .grantPoliciesForRoles(
+                projectId, List.of(custodianEmail), SetAuthGcpUserRolesStep.SNAPSHOT_GCP_IAM_ROLES);
```

Comment:
You could create this `List.of()` outside the loop and share it with both calls.